src/annotation.ts

```diff
@@ -97,7 +97,7 @@ export class Annotation extends Modifier {
         maxLeftGlyphWidth = Math.max(glyphWidth, maxLeftGlyphWidth);
         leftWidth = Math.max(leftWidth, textWidth) + Annotation.minAnnotationPadding;
       } else if (annotation.horizontalJustification === AnnotationHorizontalJustify.RIGHT) {
-        maxRightGlyphWidth = Math.max(glyphWidth, maxLeftGlyphWidth);
+        maxRightGlyphWidth = Math.max(glyphWidth, maxRightGlyphWidth);
```

Comment:
I think that you included this change by mistake.

Reply:
The code in green is what is currently in the vexflow repo.  I'm not seeing it as a change.

Reply:
Don't you see 6 files changed in this PR?

Reply:
> Don't you see 6 files changed in this PR?

yeah but 4 of them are from a merge

Reply:
I think that in order ot fix that all you need to do is:
```
git checkout master
git pull upstream master
git checkout softmax-default
git rebase master
git push -f
```
perhaps you have a different name for Mohit's repository, then you have to replace `upstream` above. Check using:

`git remote -v`

this is what I get:
```
aarondavidnewman        https://github.com/aarondavidnewman/vexflow.git (fetch)
aarondavidnewman        https://github.com/aarondavidnewman/vexflow.git (push)
origin  https://github.com/rvilarl/vexflow.git (fetch)
origin  https://github.com/rvilarl/vexflow.git (push)
ronyeh  https://github.com/ronyeh/vexflow.git (fetch)
ronyeh  https://github.com/ronyeh/vexflow.git (push)
sschmidtu       https://github.com/sschmidtu/vexflow.git (fetch)
sschmidtu       https://github.com/sschmidtu/vexflow.git (push)
upstream        https://github.com/0xfe/vexflow.git (fetch)
upstream        https://github.com/0xfe/vexflow.git (push)
wassertim       https://github.com/wassertim/vexflow.git (fetch)
wassertim       https://github.com/wassertim/vexflow.git (push)
```

Reply:
In the GitHub user interface for reviewing this PR, I see 6 files changed.

If you changed only 2 files in this PR, then ideally we would see 2 changed files (which would make it easier to review).

If you have been regularly merging your branch from 0xfe:master, shouldn't this change not appear in the PR? Or maybe you can take Rodrigo's rebase approach.

But generally for branches with involved changes, I prefer using the merge approach. I only use rebase for simple changes that can be reapplied with no (or very minimal) conflicts.